ci-helpers/postgres_setup.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+psql -U postgres -c "create extension postgis"
+psql -U postgres -q  -f ./tests/usecasesql/envirodiy/envirodiy_postgres_odm2.sql
```

Comment:
I created the tests if you look at the PR b/c I though it would be nice to separate the DB from the tests. But maybe that is not a good idea. I'll merge them.